src/main/java/se/kth/sda/skeleton/postlikes/PostLike.java

```diff
@@ -0,0 +1,56 @@
+package se.kth.sda.skeleton.postlikes;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIdentityReference;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import se.kth.sda.skeleton.posts.Post;
+import se.kth.sda.skeleton.user.User;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class PostLike {
```

Comment:
just with the naming...do we plan to use the same model for a comment like?

Reply:
yes!